src/gui/calibration_ui.py

```
@@ -0,0 +1,371 @@
+"""キャリブレーションUI実装"""
+import tkinter as tk
+from tkinter import Canvas, Toplevel
```

Ruff (0.12.2)

`tkinter.Toplevel` imported but unused

Remove unused import: `tkinter.Toplevel`

(F401)